### DISCUSSION TITLE
Just Staring out... and nothing works.

### DISCUSSION THREAD

**Discussion by stargirl245-245** [meshtastic/Discussions#44] [General]
Hey! I'm just starting out with my new Meshtastic T1000-E. And I connected it by bluetooth, my phone sees it, I connected it to my houses channel we created, and I'm unable to send any messages. Or moreover, all of them just come back with the cloud with a strike through it...
What am I doing wrong?

**Comment by garthvh** [comment]
Sounds like everything is working, there are no nodes in range

**Comment by stargirl245-245** [comment]
Turns out I was on the wrong frequency! Got it working now
